### PR TITLE
chore: add emptyDir mounted on /tmp for operator and agentk

### PR DESCRIPTION
**Pull request by seemywingz**

### CODE REVIEW

File: charts/deployment-operator/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: deployment-operator
 description: creates a new instance of the plural deployment operator
 appVersion: 0.4.32
-version: 0.4.32
+version: 0.4.33
```
Comment (michaeljguarino) [inline charts/deployment-operator/Chart.yaml:5]: we want to keep appVersion == version here, the chart will be released in the next operator release

Comment (seemywingz) [reply charts/deployment-operator/Chart.yaml:5]: okay, I noticed that, but wasn't sure because there were no code changes

Comment (michaeljguarino) [reply charts/deployment-operator/Chart.yaml:5]: You can't bump the app version because we haven't published the release.  Just leave this as is.

Comment (michaeljguarino) [reply charts/deployment-operator/Chart.yaml]: yeah that's not critical at the moment

**Review by michaeljguarino — APPROVED** [review]

**Pull request merged**